### DISCUSSION TITLE
EQ7 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#79] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/EQ7.html
Author: @sheppardw 
Primary Reviewer:

**Comment by tdegeorge** [comment]
[Activity 1.7.7](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EQ7.html#activity-rational-inequality-1)

- Part b does not have an answer in the instructor version. Is the intent for students to answer the question in interval notation or show the graph and mark where it is greater than or equal to 0? Part d asks students to take values from part b and relate them to the numerator/denominator of the combined rational function....so maybe we should ask students to write their answer in part b in interval notation?
- I'm not sure I understand the answer for part d....what do we mean by vanishes? 
- For part e, should we add "for the function g"?
- Do we want to change the ordering of the answers in part f so that the answer is not just A again (like part e)?

**Comment by tdegeorge** [comment]
Do we want to have an activity somewhere where they make a sign chart?

**Comment by tdegeorge** [comment]
Before starting with [Activity 1.7.7](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/EQ7.html#activity-rational-inequality-1), can we start with an activity where 0 is one side?

**Comment by kathypinzon** [comment]
None of the questions ask for solutions graphically on the number line. We need to include that like in EQ1 and EQ4
